LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 University of Washington, Seattle, WA
+Copyright (c) 2016-present Spryker Systems GmbH
```

Comment:
I am not sure changing license content like that is valid.
Usually you add a line here.
Legal should clarify this

Reply:
Who can I ask for clarification?

Reply:
Legal team

Reply:
Discussed with @PhilinTv. 
Let's create `beta` branch and will merge this PR into it and then will wait for clarification from legal.
But in our projects we will use this `spryker/propel-encryption-behavior:dev-beta` version until we finalize this discussion and merge it into master + make a release tag

Reply:
Answer from legal team:
We should not adapt the license, because from a legal point of view it wouldn’t be correct. Therefore pls use the original license.

Reply:
Should I revert this line?

Reply:
Yes, please.
As I expected, the license itself must remain untouched.